<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formLogin</class>
 <widget class="QDialog" name="formLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inicio de Sesion</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(64, 113, 244);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>421</width>
     <height>441</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 20px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIngresar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#btnIngresar{

	background-color: rgb(64, 113, 244);
	color: rgb(255, 255, 255);
	border-radius: 5px;

}

QPushButton#btnIngresar:hover{

	background-color: rgb(47, 84, 179);
}

QPushButton#btnIngresar:pressed{
	padding-left: 5px;
	padding-right: 5px;
	background-color: rgb(35, 64, 135);
}</string>
   </property>
   <property name="text">
    <string>Ingresar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Symbol</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Inicio Sesion</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtUsuario">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 2px solid rgba(45, 82, 101, 200);
color: rgba(0, 0, 0, 240);
padding-bottom: 7px;</string>
   </property>
   <property name="placeholderText">
    <string>Usuario</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtClave">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 0);
border: none;
border-bottom: 2px solid rgba(45, 82, 101, 200);
color: rgba(0, 0, 0, 240);
padding-bottom: 7px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Clave</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMensaje">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>281</width>

     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 0, 4);</string>
   </property>
   <property name="text">
    <string>Mensaje</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>vectorhotel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
